File: 01_Language/01_Functions/php/array_key_exists.php
<?php

$search_array = array('first' => 1, 'second' => 4, '' => false);
if (array_key_exists('first', $search_array)) {
    echo "The 'first' element is in the array";
}

if (array_key_exists(0, $search_array)) {
    echo "The '0' element is in the array";
}

File: 01_Language/01_Functions/java/src/main/java/com/onevgo/functions/ArrayKeyExists.java
package com.onevgo.functions;

import java.util.HashMap;
import java.util.Map;

public class ArrayKeyExists {
    public static <K, V> boolean arrayKeyExists(K key, Map<K, V> map) {
        return map.containsKey(key);
    }

    public static void main(String[] args) {
        Map<Object, Object> map = new HashMap<>();
        map.put("first", 1);
        map.put("second", 4);
        map.put("", false);
        System.out.println(arrayKeyExists("first", map));
    }
}
